camel-idea-plugin/src/main/java/org/apache/camel/idea/annotator/CamelSimpleAnnotator.java

```diff
@@ -49,7 +50,7 @@ boolean isEnabled() {
      */
     void validateText(@NotNull PsiElement element, @NotNull AnnotationHolder holder, @NotNull String text) {
         boolean hasSimple = text.contains("${") || text.contains("$simple{");
-        if (hasSimple && CamelIdeaUtils.isCamelSimpleExpression(element)) {
+        if (hasSimple || CamelIdeaUtils.isCamelSimpleExpression(element)) {
```

Comment:
So if do something like this in CamelIdeaUtils.isCamelSimpleExpression(element) we should be safe?

>  if (IdeaUtils.isFromJavaMethodCall(element, "simple", "log")) {
            return true;
        }